gateway/src/project.rs

```diff
@@ -687,9 +689,33 @@ where
     type Next = ProjectStarting;
     type Error = ProjectError;
 
-    async fn next(self, _ctx: &Ctx) -> Result<Self::Next, Self::Error> {
+    async fn next(self, ctx: &Ctx) -> Result<Self::Next, Self::Error> {
+        let container_id = self
```

Comment:
Use the custom `safe_unwrap!` macro for this, as it will always propagate a standard internal error for these mismatches between docker API responses and what is published in their docs.

Reply:
Nice, thanks!